### DISCUSSION TITLE
Everything Facebook revealed about the Metaverse in 11 minutes

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#156] [General]
聽聽Mark Elliot Zuckerberg怎麼詮釋元宇宙

https://youtu.be/gElfIo6uw4g

**Comment by chia-kai-chang** [comment]
另外，老高視元宇宙為奇異點，來聽聽老高的講法：https://youtu.be/hm5K-PBz0rg

**Reply by Jie2222** [reply]
老高說以後真實世界就只剩吃飯跟睡覺，滿足肉體上的需求和循環，那運動呢?有可能在虛擬世界裡達到真實的運動效果嗎?如果不行的話，以後生活連走路都不用了，整天坐著或躺著，感覺滿可怕的。

**Reply by HSU-YU-JEN** [reply]
感覺從facebook改名開始，大家就開始討論元宇宙，但感覺這一些東西都好難理解

**Comment by xCk27x** [comment]
我認為將日常生活數位化確實會非常便利，交通、開會等事情不會再那麼麻煩。甚至透過創建虛擬角色，原本會被網路所犧牲掉的課程品質也將獲得改善。
